.github/workflows/e2e.yaml

```
@@ -95,6 +95,7 @@ jobs:
           # Creating jitter so that we can stagger cluster creation to avoid throttling
           sleep $(( RANDOM % 300 + 1 ))
       - id: generate-cluster-name
+        name: Generate Cluster Name
```

I'd follow what I see elsewhere in the file which is all lowercase, or change them all to upper case :)
```suggestion
        name: generate cluster name
```

---

Also worth making the same change in e2e-upgrade if you're going to make it.